charts/zenko/values.yaml

```diff
@@ -88,3 +88,10 @@ redis:
     enabled: false
   metrics:
     enabled: true
+
+redis-ha:
+  rbac:
+    create: true
+  replicas:
+    servers: 3
```

Comment:
Hm, the more I look into this redis-ha chart and looking at this value, it sounds like a redis cluster with multiple redis sentinels as its setup. I know you asked me about this offline, and it sounded ok in practice since we introduce a single endpoint for redis. I'm not too sure the impact of using a cluster, but maybe someone else will have an answer

Reply:
This setup has (by default) 3 redis-sentinels and 3 redis-servers. The redis sentinels allow monitoring, serve notifications, start failover processes and are configuration providers for the redis-servers (the acutal keystore). The Redis documentation indicates that at least 3 sentinels are required for a robust deployment. Redis servers are under a master/slave architecture, and the master is exposed through a kuberntes service, which is the endpoint for Backbeat.

Reply:
I am a bit confused here, we must support clusters with 3 or 5 nodes. If you deploy with 5 servers, 3 is obviously not a good default setting.

How could we support the 2 types of cluster generically @NicolasT @ssalaues ? E.g. having a global value clusterSize=3 or 5 and then we could derive the proper settings for all stateful components

Reply:
This will still support 5 nodes, in those cases where we have a 5 node deployment, the main zenko/values.yaml would need to be modified for all the appropriate replicas (simply changing 3 to 5). I don't believe any other settings would need to be changed. So we inherently already support 1, 3, or 5+ node clusters thanks to the Kubernetes replica and stateful sets.

However it would be nice to have a single global value for the cluster size however I don't believe there is a way to do this (at least with a mixture of our charts and upstream charts).

Reply:
> I am a bit confused here, we must support clusters with 3 or 5 nodes. If you deploy with 5 servers, 3 is obviously not a good default setting.

Why's that? What happens when we have a 20-node K8s cluster?

We should *not* have globals depending on cluster size. It's up to the person deploying the solution to pick what's suitable. We could however have a library of `values.yml` files for various standard architectures.

Reply:
If it makes it simpler for SE's and others to deploy, it wouldn't be a bad idea to have a library of values.yml.

Reply:
so do we go for 3 as the default ?